nuxeo-features/nuxeo-platform-forms-layout-export/src/main/java/org/nuxeo/ecm/platform/forms/layout/export/LayoutResource.java

```diff
@@ -110,8 +110,14 @@ public String getAsJson(@QueryParam("layoutName") String layoutName, @QueryParam
                 }
                 if (layoutDef != null) {
                     List<WidgetDefinitionConverter> widgetConverters = service.getWidgetConverters(conversionCategory);
-                    JSONObject json = JSONLayoutExporter.exportToJson(category, layoutDef, ctx, widgetConverters);
-                    return json.toString(2);
+                    RenderingContext renderingCtx = CtxBuilder.param(LayoutExportConstants.CATEGORY_PARAMETER, category)
+                                                              .param(LayoutExportConstants.LAYOUT_CONTEXT_PARAMETER,
```

Comment:
The line formatting looks a bit strange.  Is that ok in your IDE?

Reply:
Yes, it's the IDE formatting action which produces that.